sdks/java/core/src/main/java/org/apache/beam/sdk/io/fs/ResourceIdTester.java

```diff
@@ -108,15 +108,15 @@ private static void validateFailureResolvingIds(ResourceId baseDirectory) {
     try {
       ResourceId badFile = baseDirectory.resolve("file/", RESOLVE_FILE);
       fail(String.format("Resolving badFile %s should have failed", badFile));
-    } catch (Throwable t) {
+    } catch (Exception e) {
```

Comment:
Should we narrow the exception to the one which resolve throws?

Reply:
Yea, that's a great idea. Done. I think. The test is undocumented, the error messages are a bit obscure, and the uses of the tester were already failing. But anyhow it is a more specific exception now :-)